arch/arm/arm64/include/uk/asm/lcpu.h

```diff
@@ -376,6 +376,11 @@ static inline unsigned long ukarch_read_sp(void)
 	return sp;
 }
 
+static inline void ukarch_thread_fence_acq(void)
```

Comment:
This one is no longer necessary so it can go away.

Reply:
removed